examples/remix-video-preview/app/lib/cloudinary.ts

```diff
@@ -0,0 +1,7 @@
+import { Cloudinary } from '@cloudinary/url-gen';
+
+export const cld = new Cloudinary({
+  cloud: {
+    cloudName: 'ahha'
```

Comment:
i'll swap this out after merging it so that it's not hitting your account wasting bandwidth if someone spins up the project